group_name: panda_arm
tool_name: panda_link8
approach_distance: 0.0
planning_time: 5.0
picking_configurations:
- [0.1,  2.5,   -1.3,   1.3, -1.5708, -1.5708, 0.0]
- [0.5,  2.5,   -1.3,   1.3, -1.5708, -1.5708, 0.0]
- [1.0,  2.5,   -1.3,   1.3, -1.5708, -1.5708, 0.0]
- [1.4,  2.5,   -1.3,   1.3, -1.5708, -1.5708, 0.0]

placing_configurations:
#- [0.5,  1.9, -1.5708,  -1.5708,   1.5708, -1.5708, 0.0]
- [0.1,  1.5,   -1.3,   -1.842,   1.5708, -1.5708, 0.0]
- [0.5,  1.5,   -1.3,   -1.842,   1.5708, -1.5708, 0.0]
- [1.0,  1.5,   -1.3,   -1.842,   1.5708, -1.5708, 0.0]
- [1.4,  1.5,   -1.3,   -1.842,   1.5708, -1.5708, 0.0]

#- [1.0, 4.742804391090171, -1.41123376042854, 1.2646780790192866, 1.7529467628115682, 1.569390560247801, -0.9127673651801231]
#- [1.3, 4.742804391090171, -1.41123376042854, 1.2646780790192866, 1.7529467628115682, 1.569390560247801, -0.9127673651801231]
#- [0.5, 2.3142085695077292, -2.071073904429039, -0.714737582602384, 4.329940682745921, 1.5956737813082107, -3.3416769699436775]
#- [1.0, 2.3142085695077292, -2.071073904429039, -0.714737582602384, 4.329940682745921, 1.5956737813082107, -3.3416769699436775]
#- [0.05, 2.3142085695077292, -2.071073904429039, -0.714737582602384, 4.329940682745921, 1.5956737813082107, -3.3416769699436775]


request_adapters:
- default_planner_request_adapters/AddTimeParameterization
- default_planner_request_adapters/FixWorkspaceBounds
- default_planner_request_adapters/FixStartStateBounds
- default_planner_request_adapters/FixStartStateCollision
- default_planner_request_adapters/FixStartStatePathConstraints
